src/plugin.test.ts

```diff
@@ -418,10 +418,10 @@ describe("astroRehypeRelativeMarkdownLinks", () => {
   });
 
   describe("config option validation", () => {
-    const runValidationTest = async (context, options) => {
+    const runValidationTest = async (context, options?) => {
```

Comment:
This comment actually refers to line 34 above (https://github.com/vernak2539/astro-rehype-relative-markdown-links/pull/78/files#diff-15ddc1bcd6ddf3b9483b554a2e56cad27ca25a7924fcd2e1db0bf2d7a849ccadR34) - GH doesn't allow commenting on unchanged lines :(

The old jsdoc type can be removed.  It was wrong to begin with so line 35 should actually be:

```ts
function testSetupRehype(options: { currentFilePath?: string } = {}) {
```